phlib/util.c

```diff
@@ -161,6 +161,194 @@ VOID PhCenterWindow(
     }
 }
 
+LONG PhGetDpi (
+    _In_ LONG number,
+    _In_ LONG dpi_value
```

Comment:
```suggestion
    _In_ LONG Number,
    _In_ LONG DpiValue
```

Reply:
please use pascal case, here and throughout

Reply:
I try;)